demisto_sdk/tests/integration_tests/validate_integration_test.py

```diff
@@ -357,9 +357,8 @@ def test_integration_validate_invalid_pack_path(self):
         """
         runner = CliRunner(mix_stderr=False)
         result = runner.invoke(main, [VALIDATE_CMD, "-i", AZURE_FEED_INVALID_PACK_PATH])
-        assert result.exit_code == 1
-        assert f'{AZURE_FEED_INVALID_PACK_PATH} was not found' in result.output
-        assert result.stderr == ""
+        assert 'does not exist' in result.stderr
+        assert result.exit_code == 2
```

Comment:
when are we returning exit code 2? this should be 1.

Reply:
It fails on click checking if this path exists - automatically exit code 0 (with type=click.Path(exists=True))